content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+
+4.  Transparency: The actions of the committee will be visible to
+    promote accountability, and all decision-making will be open and
+    well-documented.
+
+5.  Honesty: The committee will create an environment to promote honest
+    disclosure. All members should feel free speaking about any opinions
+    and feelings.
+
+6.  Technical Excellence: The committee acknowledges that it's
+    fundamental goal is to support diversity in technical work, and
+    members will strive to achieve excellence in this work themselves.
+
+# Committee Composition #
+
+1.  Only undergraduate cis- and trans-women and non-binary individuals
+    enrolled in computer science or a related program may serve on the
+    committee. The committtee will be comprised of the following roles:
+
+    -   Chair,
+
+    -   Secretary,
+
+    -   Finance Director,
+
+    -   Office Manager,
+
+    -   Systems Administrator,
+
+    -   Big CSters Director,
+
+    -   Director of Publicity,
+
+    -   Up to three additional positions as necessary, with titles
+        chosen by the committee.
+
+2.  The duties of the Chair are:
+
+    -   To chair all meetings;
+
+    -   To publicly represent the committee as the direct link to
+        external groups;
+
+    -   To represent the undergraduate committee at meetings of the WiCS
+        Committee;
+
+    -   To represent the undergraduate committee at meetings of any
+        other external organization; to which the commiittee is
+        accountable, when appropriate, such as the Mathematics Society;
+
+    -   To secure funding for the committee's endeavours in
+        collaboration with the Finance Director.
+
+3.  The duties of the Secretary are:
+
+    -   To take minutes at all meetings;
+
+    -   To call meetings;
+
+    -   To represent the undergraduate committee at meetings of the WiCS
+        Committee;
+
+    -   To build consensus among members in the decision-making process;
+
+    -   To ensure all members are comleting assigned resposibilities;
+
+    -   To maintain the non-financial records of the committee.
+
+4.  The duties of the Finance Director are:
+
+    -   To process financial paperwork;
+
+    -   To maintain accurate budgets and financial records;
+
+    -   To secure funding for the endeavours of the committee, in
+        collaboration with the Chair;
+
+    -   To represent the organization in a financial capacity, in all
+        relevant meetings.
+
+5.  The duties of the Office Manager are:
+
+    -   To maintain the Office Policy;
+
+    -   To train and manage office staff;
+
+    -   To ensure the office is a clean, orderly, and respectful space;
+
+    -   To create and manage the schedule of office hours;
+
+    -   To ensure the office is equipped with adequate supplies.
+
+6.  The duties of the System Administrator are:
+
+    -   To chair the Systems Committee ("wics-sys");
+
+    -   To maintain the committee's systems, including office terminals
```

Comment:
I think because this falls under the systems committee's responsibilities, it shouldn't be included here. A delegating sysadmin is not necessarily a delinquent one.